dashboard/pages/correlation/callbacks.py

```diff
@@ -118,6 +118,8 @@ def on_visualization_stage_entry(
     current_stage: int,
     concentration_value: int,
     volume_value: int,
+    activity_threshold_bottom: float,
+    activity_threshold_top: float,
```

Comment:
Imo we should have a separate callback function for this: every time one updates the thresholds, `feature_fig` also is re-generated, but there's no update wrt this one.